draft-ietf-privacypass-auth-scheme.md

```diff
@@ -348,7 +348,7 @@ authenticator using the public key of the issuer, and validate that the signed
 message matches the concatenation of the client nonce and the hash of a
 valid TokenChallenge. For context-bound tokens, origins store or reconstruct the
 contexts of previous TokenChallenge structures in order to validate the token.
-A TokenChallenge MAY be bound to a specific HTTP session with client, but
+A TokenChallenge MAY be bound to a specific TLS connection from a client, but
```

Comment:
```suggestion
A TokenChallenge MAY be bound to a specific TLS session with a client, but
```
Since it would be reasonable for the challenge to be associated with a ticket, for example.